bin/sl-run.js

```diff
@@ -12,10 +12,25 @@ var assert = require('assert');
 var config = require('../lib/config'); // May exit, depending on argv
 var log = config.logger;
 
+if (config.enableTracing) {
+  var tracer = require('../lib/tracer');
+  var traceObject = require('../lib/trace-object');
+  tracer(traceObject.tracerOptions);
+}
+
+var tracer = tracer();
+var coreTracer = tracer ? tracer.tracer : null;
+
 var agent = require('../lib/agent');
 var agentOptions = {
   quiet: config.isWorker, // Quiet in worker, to avoid repeated log messages
   logger: config.logger,
+  strongTracer: coreTracer,
+  linkMysql: coreTracer ? coreTracer.config.wrapMysql : false,
```

Comment:
No UI is currently available for the end user sets it to false.  Internally, when traceOptions.wrapMysql is set false in  lib/trace-object.js.